daemon/src/org/eclipse/iofog/field_agent/FieldAgent.java

```diff
@@ -183,7 +183,7 @@ private boolean notProvisioned() {
 				deProvision();
 			} catch (Exception e) {
 				logWarning("unable to send status : " + e.getMessage());
-				deProvision();
+//				deProvision();
```

Comment:
why did you leave this comment?

Reply:
no
maybe, it's a fix of Eugene or Maxim that i didn't merge from develop